main.go

```diff
@@ -41,9 +26,9 @@ func main() {
 
 	http.HandleFunc("/beAsked", getQuestion)
 
-	http.HandleFunc("/a", validateAnswerA)
-	http.HandleFunc("/b", validateAnswerB)
-	http.HandleFunc("/c", validateAnswerC)
+	http.HandleFunc("/validate", validateAnswer)
 
-	http.ListenAndServe(":8070", nil)
+	http.ListenAndServe(":9010", nil)
 }
+
+// preguntar a oscar si paso un slice en la query. Example localhost:9050/validate?answer=[a,b,c]&test=3?
```

Comment:
Do you need it for this though?
I mean, you can do whatever you want, I'm not going to limit you on what you do. But the current implementation is fine.